less/about/preferences.less

```diff
@@ -562,441 +376,18 @@ table.sortableTable {
-      }
-
-      .paymentsSidebarCoinbase {
-        background-image: -webkit-image-set(url(../../app/extensions/brave/img/coinbase.png) 1x,
-                                            url(../../app/extensions/brave/img/coinbase_2x.png) 2x);
-        width: 100px;
-        height: 35px;
-      }
-
-      div {
-        font-size: 12px;
-        margin: 50px 0 20px 12px;
-      }
-
-      a {
-        div {
-          margin-top: 0;
-        }
-      }
-
-      h2 {
-        font-size: 18px;
-        margin: 70px 0 -15px 12px;
-      }
-    }
-  }
-
-  // This defines the styling of the control panel (2nd) on about:preferences#payments
-  .walletBar {
-
-    background: @lightGray;
-    border-radius: @borderRadiusUIbox;
-    padding: @walletBarPadding;
-    margin: @walletBarMargin 0;
-
-    th {
-      text-align: left;
-    }
-
-    tr {
-      height: 1em;
-    }
-
-    td {
-      font-size: @fontSize;
-      padding: 10px 30px 0 0; // .settingsList .settingItem > *:not(.switchControl) 10px
-      width: auto;
-      min-width: @walletBarTableRowWidth;
-      vertical-align: top;
-
-      .settingsListContainer {
-        margin-bottom: 0;
-
-        .settingsList {
-          .settingItem {
-            .fundsAmount,
-            .fundsSelectBox,
-            .addFunds {
-              min-width: 180px;
-              width: auto;
-            }
-
-            .fundsSelectBox {
-              margin-bottom: 0;
-            }
-          }
         }
       }
     }
-
-    .settingsList .settingItem {
-
-      .paymentHistoryButton {
-        display: block;
-        font-size: @fontSize;
-        line-height: 18px;
-        color: @braveOrange;
-        height: auto;
-        margin-top: @walletBarItemMargin;
-        padding: 0;
-        text-align: left;
-        cursor: pointer;
-        white-space: normal;
-      }
-
-      .balance {
-        display: flex;
-        align-items: center;
-        vertical-align: 0;
-        height: 100%;
-        margin-bottom: 0;
-
-        span {
-          font-size: @fontSize; // for "Loading..." and .fundsFAQ
-        }
-
-        > span {
-          height: 2.25em; // .form-control
-          display: flex;
-          align-items: center;
-          margin: 0;
-          padding: 0; // .fundsFAQ
-
-          > span {
-            margin: 0; // for "Loading..."
-            padding: 0; // .fundsFAQ
-          }
-        }
-
-        + button {
-          margin-top: @walletBarItemMargin;
-        }
-
-        a {
-          &:hover {
-            text-decoration: none;
-          }
-
-          .fundsFAQ {
-            color: @gray;
-            margin: 0 0 0 5px;
-          }
-        }
-      }
-    }
-
-    .fundsAmount {
-      display: inline;
-      margin: 0;
-      width: auto;
-    }
-
-    .accountBalanceError,
-    .nextReconcileDate {
-      margin-top: @walletBarItemMargin;
-    }
-
-    .nextReconcileDate {
-      margin-bottom: 0;
-    }
   }
 }
 
+// TODO check what to remove
```

Comment:
Was there more that needed to be removed? or did you need to test anything which had been removed so far?